ghmirror/data_structures/redis_data_structures.py

```diff
@@ -67,10 +68,19 @@ def _scan_iter(self):
         """
         cursor = '0'
         while cursor != 0:
-            cursor, data = self.cache.scan(cursor)
+            cursor, data = self.wr_cache.scan(cursor)
             for item in data:
                 yield self._deserialize(item)
 
+    @staticmethod
+    def _get_connection(host='localhost'):
+        parameters = {'host': host, 'port': REDIS_PORT}
+        if REDIS_TOKEN is not None:
+            parameters['password'] = REDIS_TOKEN
+        if REDIS_SSL is not None:
+            parameters['ssl'] = REDIS_SSL
```

Comment:
This will receive the string `'True'`, not the boolean `True`.

Reply:
ah i just updated this before seeing your comment. I made it : `if REDIS_SSL == 'True'` because otherwise if would still assume an SSL connection if it was `REDIS_SSL=False`.

Reply:
Still, you want `True`, not `'True' `, in the parameters['ssl']

Reply:
ah ok, I see what you mean. It seems like it works either way, but I fixed it. Its more correct that way.

Reply:
Yep, it works because `bool('True')` is `True`. But if we ever want to set it to `False`, `bool('False')` is also `True`, so using booleans there is the right thing to do.